crates/control/src/behavior/defend.rs

```diff
@@ -58,24 +58,38 @@ impl<'cycle> Defend<'cycle> {
 
     pub fn wide_stance(
         &self,
-        wide_stance_paramters: WideStanceParameters,
+        wide_stance_parameters: WideStanceParameters,
     ) -> Option<MotionCommand> {
         let ball = self.world_state.ball?;
 
         let position = ball.ball_in_ground;
         let velocity = ball.ball_in_ground_velocity;
 
-        if velocity.x() >= wide_stance_paramters.minimum_velocity {
+        if velocity.x() >= wide_stance_parameters.minimum_velocity {
             return None;
         }
 
         let horizontal_distance_to_intersection =
             position.y() - position.x() / velocity.x() * velocity.y();
 
-        if horizontal_distance_to_intersection.abs() < wide_stance_paramters.action_radius {
-            Some(MotionCommand::WideStance)
-        } else {
-            None
+        match (
+            (-wide_stance_parameters.action_radius..=wide_stance_parameters.action_radius)
+                .contains(&horizontal_distance_to_intersection),
+            (wide_stance_parameters.action_radius..wide_stance_parameters.action_radius_left)
+                .contains(&horizontal_distance_to_intersection),
+            (-wide_stance_parameters.action_radius_left..-wide_stance_parameters.action_radius)
+                .contains(&horizontal_distance_to_intersection),
+        ) {
+            (true, _, _) => Some(MotionCommand::WideStance {
+                direction: JumpDirection::Center,
+            }),
+            (false, true, _) => Some(MotionCommand::WideStance {
+                direction: JumpDirection::Left,
+            }),
+            (false, _, true) => Some(MotionCommand::WideStance {
+                direction: JumpDirection::Right,
+            }),
+            (false, false, false) => None,
```

Comment:
I think a simple if-else statement is completely sufficient. This is basically a glorified if-else. Other opinions? @h3ndrk

Reply:
Looks like it, but not sure about the `_` in line 89 :thinking:

Reply:
I'm sure. All cases are mutually exclusive